hq/test/logic/SnykDisplayTest.scala

```diff
@@ -0,0 +1,291 @@
+package logic
+
+import model._
+import org.scalatest.{FreeSpec, Matchers}
+import utils.attempt.AttemptValues
+import scala.concurrent.ExecutionContext.Implicits.global
+
+class SnykDisplayTest extends FreeSpec with Matchers with AttemptValues {
+
+  "find organisation" in {
+    val organisation = SnykDisplay.getOrganisation(SnykDisplayTest.mockGoodOrganisationResponse, SnykOrganisationName("guardian"))
+    organisation.value shouldBe SnykOrganisation("guardian", "1111111111")
+  }
+
+  "fail to find organisation (nice)" in {
```

Comment:
What do (nice) and (not nice) mean? Will it be clear to someone else that comes to this file later when the tests are failing?

Reply:
Done.